src/lib/resource/sections/owner.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = {
+    section: {
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            required: ['owner-id', 'is-authenticated'],
+            properties: {
+                'owner-id': {
+                    type: 'string'
```

Comment:
To improve validation, lets add in the [URI format](https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-validation-01#section-7.3.5) alongside the `"type":"string"` e.g.

`"format": "uri"`